Sources/DebugDescriptions.swift

```diff
@@ -24,6 +25,15 @@ private enum Element {
     case sections([SectionViewModel])
     case registrations(Set<ViewRegistration>)
     case isEmpty(Bool)
+    case diffOnBackgroundQueue(Bool)
+    case reloadDataOnReplacingViewModel(Bool)
+    case options(CollectionViewDriverOptions)
+    case viewModel(CollectionViewModel)
+    case emptyViewProvider(EmptyViewProvider?)
+    case cellEventCoordinator(CellEventCoordinator?)
+    case scrollViewDelegate(UIScrollViewDelegate?)
+    case flowLayoutDelegate(UICollectionViewDelegateFlowLayout?)
+    case view(UICollectionView)
```

Comment:
Similarly, for `emptyViewProvider` and everything below, could we do this:

```swift
case object(label: String, value: Any?)
```

Then we could have:

```swift
case let .object(label, value):

if let value {
    buildString("\(label): \(value)", indent: indent, to: &output)
} else {
    buildString("\(label): nil", indent: indent, to: &output)
}
```

Reply:
Done.